packages/cypress/src/metadata/testPageMetadata.ts

```diff
@@ -5,18 +5,22 @@ export type TestMetaDataOptions = {
   title: string
   description: string
   baseUrl: string
+  excludeMetadata?: {
+    og?: ['image']
+    twitter?: ['image']
+  }
```

Comment:
A workaround until we add images to `createMetadata`

Reply:
Could we use the fallback image for now instead of having a workaround?